### DISCUSSION TITLE
Make field placeholders consistent and less less confusing

### DISCUSSION THREAD

**Discussion by kslstn** [tinacms/tinacms#3860] [Ideas]
Currently, the placeholder for a list of string options says "Choose an option", even if the field is not required.

This may cause content editors to believe it's preferred or even required to choose an option. In reality, the options may be used to define exceptions to the (usually preferred) default, that's applied when the field remains empty.

A quick fix could be removing placeholders altogether, as now AFAIK they're only used in lists.

In my opinion, if placeholders are even desired, they should be added to all field types. I'd prefer to have no placeholders for fields that are optional.  For instance the placeholder for an options list should say "Choose an option" if the the field is required. There should be no placeholder when the field is not required.

**Comment by logan-anderson** [comment]
@kslstn  Do you mean something like this?

What if for 

`required: true` =>  "Choose an option" (how it is now)

and 

`required: false` =>  "No Selection"

**Comment by kslstn** [comment]
Yes, I love it!
